tests/python_helics/test_message_federate.py

```diff
@@ -51,6 +51,7 @@ def mFed():
     while (h.helicsBrokerIsConnected(broker)):
         time.sleep(1)
 
+    h.helicsFederateInfoFree(fedinfo)
```

Comment:
you aren't double freeing here are you?    otherwise this seems a little inconsistent with the other tests?

Reply:
Why is this a double free? I think this is identical to what we are doing in the other Python tests.

Reply:
It looks like in some places you call a separate free function ' FreeFederate(fFed, ffedinfo)'  that frees the fed info, but I wasn't sure how that interacted with this free?  It may not, I am just trying to clarify things.

Reply:
Yes, this FreeFederate calls helicsFederateInfoFree, so it's all good. Only the `test_message_filter.py` using FreeFederate because it made it easier to create multiple federates in a single test this way.